test/extended/apiserver/root_403.go

```diff
@@ -25,7 +25,7 @@ var _ = g.Describe("[sig-api-machinery][Feature:APIServer]", func() {
 
 	oc := exutil.NewCLI("apiserver")
 
-	g.It("anonymous browsers should get a 403 from /", func() {
+	g.It("anonymous browsers should get a 403 from / [apigroup:config.openshift.io]", func() {
```

Comment:
this test should be rewritten to avoid the dep

Reply:
Removed